lib/protocol/protocol.js

```diff
@@ -273,6 +273,9 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
       if (parsedDriverRes) {
         isW3C = parsedDriverRes.isW3C;
         isMJSONWP = parsedDriverRes.isMJSONWP;
+        if (driverRes.error) {
```

Comment:
driverRes or parsedDriverRes?

Reply:
They would both have the same error, but I'll change it to `parsedDriverRes` because that's more obvious.